src/mlpack/core/optimizers/sgd/update_policies/nesterov_momentum_update.hpp

```diff
@@ -0,0 +1,127 @@
+/**
+ * @file nesterov_momentum_update.hpp
+ * @author Sourabh Varshney
+ *
+ * Nesterov Momentum Update for Stochastic Gradient Descent.
+ *
+ * mlpack is free software; you may redistribute it and/or modify it under the
+ * terms of the 3-clause BSD license.  You should have received a copy of the
+ * 3-clause BSD license along with mlpack.  If not, see
+ * http://www.opensource.org/licenses/BSD-3-Clause for more information.
+ */
+#ifndef MLPACK_CORE_OPTIMIZERS_SGD_NESTEROV_MOMENTUM_UPDATE_HPP
+#define MLPACK_CORE_OPTIMIZERS_SGD_NESTEROV_MOMENTUM_UPDATE_HPP
+
+#include <mlpack/prereqs.hpp>
+
+namespace mlpack {
+namespace optimization {
+
+/**
+ * Nesterov Momentum update policy for Stochastic Gradient Descent (SGD).
+ *
+ * Learning with SGD can be slow. Applying Standard momentum can accelerate
+ * the rate of convergence. Nesterov Momentum application can accelerate the
+ * rate of convergence to O(1/k^2).
+ * 
+ * @code
+ * @techreport{Nesterov1983,
+ *   title       = {A Method Of Solving A Convex Programming Problem With 
+ * 					Convergence Rate O(1/K^2)},
+ *   author      = {Yuri Nesterov},
+ *   institution = {Soviet Math. Dokl.},
+ *   volume      = {27},
+ *   year        = {1983},
+ *   url         = {http://www.cis.pku.edu.cn/faculty/vision/zlin/1983-A%20
```

Comment:
This is not an offical url, can you remove the reference here.

Reply:
I am not finding any other url related to this. Can you suggest one?

Reply:
Let's just remove the url tag.

Reply:
Removed